keyboards/clueboard/66_hotswap/rules.mk

```diff
@@ -10,14 +11,14 @@ OPT_DEFS += -DBOOTLOADER_SIZE=4096
 # Build Options
 #   comment out to disable the options.
 #
-BOOTMAGIC_ENABLE = no 	# Virtual DIP switch configuration(+1000)
-MOUSEKEY_ENABLE = no	# Mouse keys(+4700)
-EXTRAKEY_ENABLE = yes 	# Audio control and System control(+450)
-CONSOLE_ENABLE = yes	# Console for debug(+400)
-COMMAND_ENABLE = yes	# Commands for debug and configuration
-NKRO_ENABLE = yes	# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
-AUDIO_ENABLE = no
-RGBLIGHT_ENABLE = yes	# Enable keyboard underlight functionality
-MIDI_ENABLE = no 	# MIDI controls
-UNICODE_ENABLE = no 	# Unicode
-BLUETOOTH_ENABLE = no	# Enable Bluetooth with the Adafruit EZ-Key HID
+BOOTMAGIC_ENABLE = no
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = yes
+CONSOLE_ENABLE = yes
```

Comment:
disabling the console will also disable debugging and printing, and save a lot of space, too.

Reply:
It will, but for a default firmware I want that enabled, sometimes the debugging is helpful.

Reply:
Point.